org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsi/XSIFormatterLegacyTest.java

```diff
@@ -0,0 +1,187 @@
+				"    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + //
+				"    xmlns:util=\"http://www.springframework.org/schema/util\"\r\n" + //
+				"    xsi:schemaLocation=" + //
+				"</beans>";
+		assertFormat(content, content, settings);
+
+		content = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<beans\r\n" + //
+				"    xmlns=\"http://www.springframework.org/schema/beans\"\r\n" + //
+				"    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + //
+				"    xmlns:util=\"http://www.springframework.org/schema/util\"\r\n" + //
+				"    xsi:schemaLocation=\"" + //
+				"</beans>";
+		assertFormat(content, content, settings);
+
+		content = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<beans\r\n" + //
+				"    xmlns=\"http://www.springframework.org/schema/beans\"\r\n" + //
+				"    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + //
+				"    xmlns:util=\"http://www.springframework.org/schema/util\"\r\n" + //
+				"    xsi:schemaLocation=\"\"" + //
+				"</beans>";
+		assertFormat(content, content, settings);
+	}
+
+	@Test
+	public void xsiSchemaLocationSplitOnElementWithTabs() throws BadLocationException {
+		SharedSettings settings = createSettings();
+		settings.getFormattingSettings().setTabSize(4);
+		settings.getFormattingSettings().setInsertSpaces(false);
+		XSISchemaLocationSplit.setSplit(XSISchemaLocationSplit.onElement, settings.getFormattingSettings());
+		String content = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<beans\r\n" + //
+				"    xmlns=\"http://www.springframework.org/schema/beans\"\r\n" + //
+				"    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + //
+				"    xmlns:util=\"http://www.springframework.org/schema/util\"\r\n" + //
+				"    xsi:schemaLocation=\"\r\n" + //
+				"        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd\r\n"
+				+ //
+				"        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd\">\r\n"
+				+ //
+				"\r\n" + //
+				"</beans>";
+		String expected = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<beans\r\n" + //
+				"		xmlns=\"http://www.springframework.org/schema/beans\"\r\n" + //
+				"		xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + //
+				"		xmlns:util=\"http://www.springframework.org/schema/util\"\r\n" + //
+				"		xsi:schemaLocation=\"http://www.springframework.org/schema/beans\r\n" + //
+				"							http://www.springframework.org/schema/beans/spring-beans.xsd\r\n" + //
+				"							http://www.springframework.org/schema/util\r\n" + //
+				"							http://www.springframework.org/schema/util/spring-util.xsd\">\r\n" + //
+				"\r\n" + //
+				"</beans>";
+		assertFormat(content, expected, settings);
+	}
+
+	private static SharedSettings createSettings() {
+		SharedSettings settings = new SharedSettings();

```

Comment:
Is this class for the new formatter or the old formatter? The name suggest old formatter, but I think this `SharedSettings` uses the new formatter.